jekyll/_posts/guides/2018-09-21-usage-of-the-matrix-js-sdk.md

```diff
@@ -0,0 +1,198 @@
+---
+layout: post
+categories: guides
+---
+
+<style>
+ul {
+    list-style: circle;
+    margin-left: 30px;
+}
+p {
+    padding-bottom: 1em;
+    line-height: 1.7em;
+}
+h2 {
+    margin-bottom: 15px;
+    margin-top: 30px;
+}
+
+</style>
+
+# Usage of the matrix-js-sdk
+
+Let's explore how we would make a very simple Matrix client, with only the ability to perform an initial sync, and to get member lists and the timeline for rooms of our choice.
+
+This article will explore the [Matrix Client-Server API], making use of the [matrix-js-sdk]. Later articles may discuss making the underlying calls. Specifically we will cover:
+
+* login
+* simple syncing
+* listening for timeline events
+* access the `MatrixInMemoryStore`
+* sending messages to rooms
+
+We'll use Node.js as our environment, though the [matrix-js-sdk] can also be used directly in the browser.
+
+## Let's write JavaScript
+
+The first thing to do is install the [matrix-js-sdk] from NPM:
+
+```unix
+npm install matrix-js-sdk
+```
+
+We include the SDK in our source exactly as expected:
+
+```javascript
+var sdk = require("matrix-js-sdk");
```

Comment:
Probably makes sense to have this in ES6 now so this should probably be `import sdk from 'matrix-js-sdk'`